### DISCUSSION TITLE
Debug with Visual Studio Code and native JavaScript Debugger

### DISCUSSION THREAD

**Discussion by conduent-teamvad** [vuepress-theme-hope/vuepress-theme-hope#2944] [Q&A] [answered]
Hi,

I used to be able to debug the JS code by:
- simply opening a `JavaScript Debug Terminal` 
- running `npm run docs:dev` 
- and setting a break point where I needed (ex: in the `config.js` > `onPrepared` event)

This was before I started using this theme.

Why can't I do now and how can I do it?

Thanks.

**Comment by Mister-Hope** [comment, marked as answer]
The config file is loaded by esbuild, which means the application does no actually import this file, so breakpoint won't work.

I believe same thing will happens on ts config file with vite / vitest and so on.

If you want further discussion, go to vuepress/vuepress-next repo. This is not something related to VuePress Theme Hope.
